wright/src/grammar/parsers/statement/expression_statement.rs

```diff
@@ -0,0 +1,35 @@
+use crate::grammar::ast::{Expression, ExpressionSt};
+use crate::grammar::model::{HasFragment, Fragment};
+use nom::IResult;
+use nom::character::complete::{char as ch, multispace0};
+use nom::combinator::{map, recognize};
+use nom::sequence::{preceded, terminated};
+
+impl<'s> ExpressionSt<'s> {
+    fn inner(frag: Fragment<'s>) -> IResult<Fragment<'s>, Expression<'s>> {
+        terminated(
+            Expression::parse,
+            preceded(
+                multispace0,
+                ch(';'),
+            ),
```

Comment:
use `delimited` insead of `preceded`. We should also parse through following white space

Reply:
Do we want to write parsers as 
```
<whitespace> <subobj> <subobj> <whitespace>
```
or as
```
<subobj> <whitespace> <subobj>
```
If we stay consistent we can pick either one.